components/experimental/src/dimension/currency/format.rs

```diff
@@ -7,15 +7,16 @@ use fixed_decimal::FixedDecimal;
 use icu_decimal::FixedDecimalFormatter;
 use writeable::Writeable;
 
-use crate::dimension::currency::formatter::CurrencyCode;
 use crate::dimension::currency::options::CurrencyFormatterOptions;
 use crate::dimension::currency::options::Width;
 use crate::dimension::provider::currency;
 use crate::dimension::provider::currency::CurrencyEssentialsV1;
 
+use crate::dimension::currency::CurrencyCode;
+
 pub struct FormattedCurrency<'l> {
     pub(crate) value: &'l FixedDecimal,
-    pub(crate) currency_code: CurrencyCode,
+    pub(crate) currency_code: &'l CurrencyCode,
```

Comment:
`CurrencyCode` is `TinyAsciiStr<3>`, which is smaller than a reference. Why change this to borrow?

Reply:
you are right, even the architecture is 32-bit still less or the same.